packages/@aws-cdk/aws-ec2/lib/security-group.ts

```diff
@@ -31,20 +67,26 @@ export abstract class SecurityGroupBase extends Construct implements ISecurityGr
    */
   public readonly defaultPortRange?: IPortRange;
 
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    Object.defineProperty(this, isSecurityGroupSymbol, { value: true });
```

Comment:
Is that the cleanest way in TypeScript to create symbol property? I think we should be able to do something like:

```ts
class Foo {
  private readonly [isSecurityGroupSymbol] = true
}
```

Reply:
Doing that leads to an "unused variable" error, this doesn't. Also, this makes the property nonenumerable, which I kinda like.